content/Watsonx/WatsonxData/109.md

```diff
@@ -58,6 +58,8 @@ In this example scenario, you’re going to “move” the **sls_product_line_lo
 
 3. In the **Create schema** pop-up window, select/enter the following information, and then click the **Create** button:
 
+> **Note:** If you using shared COS bucket created for this workshop, prefix your schema name with your initials to avoid object conflict issue in bucket. 
```

Comment:
I think this note is all we really need.  Minor wording update:

> If you are using the shared COS bucket provided for your workshop, prefix the schema name with your initials or some other uniqueid value to avoid naming conflicts in the COS bucket.